### DISCUSSION TITLE
Accuracy Assessment: Saving test ROIs to SCP file

### DISCUSSION THREAD

**Discussion by lukaswimmer** [semiautomaticgit/SemiAutomaticClassificationPlugin#185] [Q&A] [answered]
Hello, 

After processing a classification, I like to do the accuracy assessment with SCP. Following the tutorial/manual/youtube video mentioned on your website, I successfully created the ROIs. Assigning the correct class to the MC ID and the C ID also works. In case I have finished assigning values, how do I save these changes to the SCP file? 

If I click "Save", a new ROI in a new line is created. Instead, I like to only have my changes saved. Automatic saving does not work, if I open the SCP file in a new QGIS session, it is empty. 

When looking at your tutorials, I have the impression that you do not save at all and directly start the accuracy assessment after assigning the values. Unfortunately, I need to have a copy of the ROIs for documentation reasons and because I work in an environment with unregular power outages.

I work with SCP 7.9.9 on QGIS 3.16.11.

Thanks for your reply.

**Comment by semiautomaticgit** [comment, marked as answer]
Hello @lukaswimmer ,

ROIs should be automatically saved in the .scp file if the option "Autosave" is active (https://semiautomaticclassificationmanual.readthedocs.io/en/latest/scp_dock.html#roi-signature-list).
Also, ROIs are saved whenever the QGIS project is saved.
If the ROIs are never saved please copy the log file following these steps https://semiautomaticclassificationmanual.readthedocs.io/en/latest/faq.html#how-can-i-report-an-error

**Reply by lukaswimmer** [reply]
Hello @semiautomaticgit 

Thanks for your solutions.

I am sorry for the late reply; saving the QGIS project file works. I did not realize that because I usually do not work with a project file. Nevertheless, the automatic ROI saving did not work. Currently, I do not work with classifications - I will post the log the next time I experience that error.
